getting_started/step_by_step/singletons_autoload.rst

```diff
@@ -116,7 +116,7 @@ it inherits from ``Node``:
 
 The next step is to add this script to the autoLoad list. Open
 ``Project > Project Settings`` from the menu, switch to the "AutoLoad" tab and
-select the script by clicking the ``..`` button or typing its path:
+select the script by clicking the browse button (|Browse|) or typing its path:
```

Comment:
I wouldn't add icons in the docs, unless this is done in a systematic manner everywhere (which is not the case, this is the first attempt at adding a given icon to the docs).

IMO mentioning the "Browse" button is enough, but it could also be outlined in red in the following screenshot.

BTW that screenshot needs to be fixed, as it has a typo `Gloabl.gd`.

Reply:
[Introduction to the 2D animation features](https://docs.godotengine.org/en/latest/tutorials/animation/introduction_2d.html) uses icons in a few places, that's what I based my changes on.

Reply:
Fair enough then :)